sympy/physics/units/util.py

```diff
@@ -120,21 +123,24 @@ def get_total_scale_factor(expr):
     return expr_scale_factor * Mul.fromiter((1/get_total_scale_factor(u) * u) ** p for u, p in zip(target_units, depmat))
 
 
-def quantity_simplify(expr):
+def quantity_simplify(expr, across_dimensions: bool=False, unit_system="SI"):
```

Comment:
I would suggest to go as far as not assuming a default unit system. That is, requiring the users to specify the unit system they want to use. The purpose of the _units_ module is to be useful for generic unit systems and I would attempt at distancing from _SI_ as much as possible.

Maybe `unit_system: Optional[Union[UnitSystem, str]] = None` and then raise an exception if `across_dimensions=True` and `unit_system` is `None`?

Reply:
Other functions that take `unit_system` assume a default value of `"SI"`, like `convert_to()`. I think it would be better to at least remain consistent.

Reply:
What about uniting the parameters `across_dimensions` and `unit_system`?

Reply:
> Other functions that take `unit_system` assume a default value of `"SI"`, like `convert_to()`

I regret that decision.

Reply:
Uniting the parameters sounds confusing from an end user's perspective. Should this feature be moved out into a separate function? What should it be called? Something like `quantity_unify()`?

Reply:
> Uniting the parameters sounds confusing from an end user's perspective.

Probably, it was just an idea to avoid having too many parameters.